### PR TITLE
Server status

### PR DESCRIPTION
**Pull request by j-meneghini**

### CODE REVIEW

File: dashboard/static/dashboard/js/monitor.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  tbl.appendChild(tbdy);
+  document.getElementById("statusInfo").appendChild(tbl);
+  setTimeout(tableCreate, 60000);
+  setTimeout(time, 1000);
+}
+
+// Websites monitored url
+var siteUrl = [
+  'https://csunplugged.org/en/',
+  'http://csfieldguide.org.nz/',
```
Comment (JackMorganNZ) [inline dashboard/static/dashboard/js/monitor.js]: Should be:

```suggestion
  'https://csfieldguide.org.nz/en/',
```

File: dashboard/static/dashboard/js/monitor.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  'https://www.dthm4kaiako.ac.nz/'
+]
+
+// Website monitored names
```
Comment (JackMorganNZ) [inline dashboard/static/dashboard/js/monitor.js]: Would be great to have this list and the list above merged in an dictionary, or a list of lists. Something where the name and URL are grouped.

File: dashboard/static/dashboard/js/monitor.js (hunk 1 above)
Comment (j-meneghini) [reply dashboard/static/dashboard/js/monitor.js]: Done

File: dashboard/static/dashboard/js/monitor.js (hunk 2 above)
Comment (j-meneghini) [reply dashboard/static/dashboard/js/monitor.js]: Done.

**Pull request merged**